test/e2e/areas/action-bar/editor-action-bar.test.ts

```diff
@@ -152,3 +127,66 @@ async function verifyLineNumbersVisibility(page, isVisible: boolean) {
 		isVisible ? await lineNumbers.toBeVisible() : await lineNumbers.not.toBeVisible();
 	}
 }
+
+async function verifyOpenChanges(page: Page) {
+	await test.step('verify "open changes" shows diff', async () => {
+
+
+		// make change & save
+		await page.getByText('date', { exact: true }).click();
+		await page.keyboard.press('X');
+		await bindPlatformHotkey(page, 'S');
+
+		// click open changes & verify
+		await page.getByLabel('Open Changes').nth(1).click();
+		await expect(page.getByLabel('Revert Block')).toBeVisible();
+		await expect(page.getByLabel('Stage Block')).toBeVisible();
+		await page.getByRole('tab', { name: 'quarto_basic.qmd (Working' }).getByLabel('Close').click();
+
+		// undo changes & save
+		await bindPlatformHotkey(page, 'Z');
+		await bindPlatformHotkey(page, 'S');
+	});
+}
+
+async function bindPlatformHotkey(page: Page, key: string) {
+	await page.keyboard.press(process.platform === 'darwin' ? `Meta+${key}` : `Control+${key}`);
+}
+
```

Comment:
seems like we may need a new place to start collecting things like this.  i.e. general purpose helper functions.  not necessary for this PR but maybe keep in mind that we might want a positronCode.ts or something

Reply:
(and that might be a good step towards getting unhooked from the MS dependencies)
Would be great if everything didn't depend on code.ts since code.ts is loaded with getElements, etc
argh